*graalpython.lib-python.3.test.test_normalization.NormalizationTest.test_bug_834676
